docs/source/howto/share_data.rst

```diff
@@ -0,0 +1,248 @@
+
+    $ verdi export create my-calculations.aiida --nodes 12 123 1234
+
+As usual, you can use any identifier (label, PK or UUID) to specify the nodes to be exported.
+
+The resulting archive file ``my-calculations.aiida`` contains all information pertaining to the exported nodes.
+The default traversal rules make sure to include the complete provenance of any node specified and should be sufficient for most cases.
+See ``verdi export create -h`` for ways to modify the traversal rules.
+
+.. tip::
+
+    If you want to make sure that the archive includes everything you intended, you can create a new profile and import it:
+
+    .. code-block:: console
+
+            verdi quicksetup --profile test-export  # create new profile
+            verdi --profile test-export import my-calculations.aiida
+
+            # now use e.g. the QueryBuilder to query the database
+
+Please remember to use **UUIDs** when pointing your colleagues to data *inside* an AiiDA export file, since UUIDs are guaranteed to be universally unique (while PKs aren't).
+
+Using groups for data exporting
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If the number of results to be exported is large, for example in a high-throughput study, use the ``QueryBuilder`` to add the corresponding nodes to a group ``my-results`` (see :ref:`how-to:data:organize:group`).
+Then export the group:
+
+.. code-block:: console
+
+    $ verdi export create my-calculations.aiida --groups my-results
+
+Publishing AiiDA export files
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+AiiDA export files can be published on any research data repository, for example the `Materials Cloud Archive`_, `Zenodo`_, or the `Open Science Framework`_.
+When publishing AiiDA export files on the `Materials Cloud Archive`_, you also get an interactive *EXPLORE* section, which allows peers to browse the AiiDA provenance graph directly in the browser.
+
+
+.. _Zenodo: https://zenodo.org
+.. _Open Science Framework: https://osf.io
+.. _Materials Cloud Archive: https://archive.materialscloud.org
+
+
+Importing an archive
+^^^^^^^^^^^^^^^^^^^^
+
+Use ``verdi import`` to import AiiDA archives into your current AiiDA profile.
+``verdi import`` accepts URLs, e.g.:
+
+.. code-block:: console
+
+    $ verdi import "https://archive.materialscloud.org/record/file?file_id=2a59c9e7-9752-47a8-8f0e-79bcdb06842c&filename=SSSP_1.1_PBE_efficiency.aiida&record_id=23"
+
+During import, AiiDA will avoid identifier collisions and node duplication based on UUIDs (and email comparisons for ``User`` entries).
+By default, existing entities will be updated with the most recent changes.
+Node extras and comments have special modes for determining how to import them - for more details, see ``verdi import -h``.
+
+.. tip:: The AiiDA archive format has evolved over time, but you can still import archives created with previous AiiDA versions.
+    If an outdated archive version is detected during import, you will be prompted to confirm automatic migration of the archive file.
```

Comment:
Does this work with multi-line + in-line?

Reply:
thanks, I checked and it works fine.